app/views/patients/update.js.erb

```diff
@@ -49,6 +49,16 @@ if($('#row-<%= @patient.id %>')) {
   );
 }
 
+// Update the archive date on audit
+$('#patient_fulfillment_audited')
```

Comment:
Lots of parent scanning here! Does `$('p.archive-note').text('stuff');` not cooperate?